script/input/1/aave.json

```diff
@@ -0,0 +1,15 @@
+{
+    "chainlog": "0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F",
+    "admin": "0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB",
```

Comment:
Do we need a key for admin or we can use always the pause proxy from the chainlog?

Reply:
I think it's helpful to not assume pause proxy here. It's possible someone wants to make a deploy under an EOA for testing purposes or something.